tests/testDoubleGaussianPsf.py

```diff
@@ -34,14 +34,8 @@
 import lsst.afw.display.utils as displayUtils
 import lsst.meas.algorithms as measAlg
 import lsst.pex.exceptions as pexExceptions
-import lsst.pex.logging as logging
 import lsst.utils.tests
 
-try:
-    type(verbose)
-except NameError:
-    verbose = 0
-    logging.Trace_setVerbosity("algorithms.Interp", verbose)
```

Comment:
Similar to ``testInterp.py``, there are debug messages from ``afw.math`` but no loggers of these names exist.